openpype/lib/build_template.py

```diff
@@ -0,0 +1,61 @@
+from openpype.pipeline import registered_host
```

Comment:
Can we move this to `openpype.pipeline`? We're in process of reducing pipeline related functions from `openpype.lib` to avoid cross imports.

Reply:
This looks like a package of workfile template build. So maybe a folder with this file, `abstract_template_loader.py` and build_template_exceptions.py`?

Reply:
Should I put the file `build_template.py`, `abstract_template_loader.py` and `build_template_exceptions.py`
in `openpype/lib/build_workfile` or `openpype/build_workfile`?

Reply:
@iLLiCiTiT Can you tell me which folder please.

Reply:
I'm sorry for late response, just `./openpype/pipeline`. Keep updated changes in `avalon_context.py` most of those functions were changed or removed over time.